distributed/cli/tests/test_dask_worker.py

```diff
@@ -723,11 +723,11 @@ def test_error_during_startup(monkeypatch, nanny, loop):
                 assert worker.wait(10) == 1
 
 
-@gen_cluster(nthreads=[], client=True)
-async def test_single_executable_deprecated(c, s):
-    with popen(["dask-worker", s.address], capture_output=True) as worker:
-        # ensure deprecation warning is emitted
-        wait_for_log_line(b"FutureWarning: dask-worker is deprecated", worker.stdout)
+def test_single_executable_deprecated():
+    assert (
+        b"FutureWarning: dask-worker is deprecated"
+        in subprocess.run(["dask-worker"], capture_output=True).stderr
```

Comment:
Just a note for other readers: this looks strange because we're not providing a scheduler address for the worker, but is okay in this case because we just want to check that this warning is emitted (which still happens when if the `dask-worker` command fails)